privacy.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: page
+title: Data Privacy
+tagline: Data Privacy information for Jupyter Related software
+permalink: /privacy
+---
+
+
+# F.A.Q
+
+This page tried to answer regularly asked question about Jupyter and Data Privacy.
+
+## Does Project Jupyter **Software** Collect information ? 
+
+Project Jupyter Software does not collect any information about its installation or usage. 
+
+All the software that is developed by the Jupyter Project are free of tracking
+code. When you install and deploy Jupyter add related libraries on your own
+computer or servers no informations is collected not exchanged with the Jupyter
+Developer.
+
+We can only guaranty this is true when you use Jupyter Directly and not via a
+Third Party Vendor that provide hosted Jupyter Notebook, if you have questions
+about hosted Jupyter please see your vendor documentation and privacy sections.
```

Comment:
```suggestion
Jupyter's privacy policy does not apply to third-party vendors that host Jupyter
products for remote or cloud usage. If you have questions about hosted Jupyter
products, please consult your vendor's documentation and privacy policy.
```

The word _guaranty_/_guarantee_ may establish a legal expectation, which we should avoid.